Makefile

```diff
@@ -33,8 +33,10 @@ validate:
 	echo "## Validate" ;\
 	echo "##" ;\
 	if [[ ! -z "${CI}" ]]; then \
```

Comment:
`-n` is true if not empty, fyi. And why the `CI?` if not empty?

Reply:
CI was a var on GitHub, and doesn't `-z` check existance?

Reply:
I mean more, why are you echoing out `"CI?"`. The `-z` evaluates to true if the string is empty. Since you are flipping that conditional anyway, you can just go with `-n`.

Reply:
so just `-n` eval